.github/workflows/generate-jvm-extension.yaml

```diff
@@ -0,0 +1,82 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name: Generate JVM Extensions
+
+on:
+  workflow_dispatch:
+    inputs:
+      artifactIds:
+        description: 'Comma separated list of Camel component artifact ids'
```

Comment:
An example would be handy to see whether they should be with or without the `camel-` prefix.

Reply:
It can be with or without, as the `camel-` prefix is stripped out. But I can tweak the description so that it's obvious.

Reply:
Nifty!